app/decorators/min_commission.go

```diff
@@ -51,37 +62,13 @@ func (min MinCommissionDecorator) AnteHandle(
 		return nil
 	}
 
-	validAuthz := func(execMsg *authz.MsgExec) error {
```

Comment:
? - This is needed to ensure a validator does not do an Authz message to bypass the minimum commission amount

Or else people can bypass with 0 commission through an authz create new validator msg

Reply:
100%.  Thank you.

Reply:
will revert this file for now.  When I do that I'll let ya know why that happened.